tests/test_build_docs_version.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+import pytest
+
+from build_docs import Version
+
+
+def test_equality() -> None:
```

Comment:
Are all the `-> None` necessary?

Reply:
Yeah, we can remove them if you prefer, I don't mind too much either way. But when we add mypy, let's add them back if it complains about it.

Reply:
Adding them opt-in the function to be checked by mypy, so it will be usefull if we use mypy on this repo sometime.